sandbox/src/main/java/com/bumble/appyx/sandbox/client/integrationpoint/IntegrationPointExampleNode.kt

```diff
@@ -34,8 +35,8 @@ import java.util.*
 class IntegrationPointExampleNode(buildContext: BuildContext) : Node(buildContext = buildContext),
     RequestCodeClient {
 
-    private val permissionRequester = integrationPoint.permissionRequester
-    private val activityStarter = integrationPoint.activityStarter
+    private val permissionRequester: PermissionRequester get() = integrationPoint.permissionRequester
```

Comment:
Having it as a getter here might make it more obvious to developers looking at the example that you shouldn't access it within the constructor.

Otherwise we could remove these variables and simply inline the calls to `integrationPoint.permissionRequester`